protocol/pom.xml

```diff
@@ -149,8 +152,12 @@
           <plugin>
             <groupId>com.mycila</groupId>
             <artifactId>license-maven-plugin</artifactId>
+            <configuration>
```

Comment:
Do no force `<skip>`. It should be already `false`.

Reply:
Thanks for catching. Removed, verified that `mvn process-sources -P proto-compile` and `mvn clean install -P proto-compile` generated code with the headers and that `mvn process-sources -P proto-compile -Dlicense.skip=true` generates sources without license headers.